styles.css

```diff
@@ -684,3 +712,413 @@ footer {
+    color: white;
+    background-image: url(images/background.png);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    background-size: 34%;
+    width: 100%;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+  }
+
+  #main > div {
+    width: none;
+    max-width: 1440px;
+    margin-left: -5%;
+  }
+
+  h1 {
+    width: 100%;
+    height: 124px;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #ff6b00;
+    margin-top: 20px;
+  }
+
+  br {
+    display: inline;
+  }
+
+  #h2 {
+    width: 100%;
+    height: 24px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+  }
+
+  .data-arc {
+    width: 100%;
+    height: 72px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 24px;
+    color: #f4f5f7;
+    margin-left: 20px;
+  }
+
+  .social-media {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 190px;
+    width: auto;
+    margin-right: 5%;
+    align-self: center;
+  }
+
+  .just-items {
+    padding: 0 120px 0;
+  }
+
+  .pic-ajust {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 24px;
+  }
+
+  .work-story {
+    padding-top: 0;
+  }
+
+  /* --------------------about me section -------------- */
+
+  .about-me {
+    background: none;
+  }
+
+  .tech-skills {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 90px 0;
+  }
+
+  .about-header-txt {
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #ff6b00;
+    width: 375px;
+    padding: 40px 0 0 0;
+  }
+
+  .about-txt {
+    width: 566px;
+    height: 148px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  /* -------------------form area-------------- */
+
+  #form-area1 > div {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    justify-items: center;
+  }
+
+  .form-txt {
+    width: 90%;
+    height: 256px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #262626;
+    padding-left: 16px;
+    text-align: start;
+  }
+
+  .social-media1 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    position: relative;
+    width: 200px;
+    height: 40px;
+    left: 433px;
+    top: 62px;
+    margin-left: 362px;
+  }
```

Comment:
- [ ] Please I kindly suggest sticky to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-1278&t=m82U1CnKG1ogD0DD-0)

_**REQUIRED FIGMA DESIGN**_

![Screenshot (137)](https://user-images.githubusercontent.com/78104544/229001050-e19b6a3b-c895-4958-92c8-b579ac49302b.png)

_**YOUR DESIGN**_

- [ ] Please I highly suggest checking `marging-left` property on the `.social-media1` class for the icons to fit the Figma design. 😃 

![Screenshot (140)](https://user-images.githubusercontent.com/78104544/229001404-54248bd0-61ec-4e55-a385-259c7a8ef1fc.jpg)

Reply:
resolved

Reply:
[braincee](https://github.com/braincee) 

hi can you check my code please